tests/func/test_import.py

```diff
@@ -330,7 +346,7 @@ def test_pull_no_rev_lock(erepo_dir, tmp_dir, dvc):
 
 def test_import_from_bare_git_repo(
     tmp_dir, make_tmp_dir, erepo_dir, local_cloud
-):
+):  # pylint:disable=unused-argument
```

Comment:
I don't think these changes are needed. You probably were missing some dependencies for pylint or something.

Reply:
It's annoying, I just don't know how to fix this `pylint` error. I'll try to solve it later.

Reply:
@karajan1001 It only shows up locally? Try re-installing `pip install '.[all,tests]'`.